test/spec/controllers/exportimport.js

```diff
@@ -210,7 +203,7 @@ describe('Controller: ExportImportCtrl', function () {
 
     scope.createExportFile( scope.selectedExports );
 
-    expect(scope.downloadLink.indexOf('blob:http://localhost:8080/')).to.be.above(-1);
+    expect(scope.downloadLink.indexOf('')).to.be.above(-1);
```

Comment:
When running the tests, the createExportFile method fails with the error "browser not capable of creating blob", hence no download link.  So will have to reconsider this.  Thoughts?